[2024-04-08T05:06:50.445+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-14T00:00:00+00:00 [queued]>
[2024-04-08T05:06:50.459+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-14T00:00:00+00:00 [queued]>
[2024-04-08T05:06:50.459+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:50.460+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2024-04-08T05:06:50.460+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:50.529+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): second_task> on 2021-11-14 00:00:00+00:00
[2024-04-08T05:06:50.534+0000] {standard_task_runner.py:55} INFO - Started process 9258 to run task
[2024-04-08T05:06:50.537+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-11-14T00:00:00+00:00', '--job-id', '1210', '--raw', '--subdir', 'DAGS_FOLDER/our_firest_dag.py', '--cfg-path', '/tmp/tmpbb8ysupw']
[2024-04-08T05:06:50.539+0000] {standard_task_runner.py:83} INFO - Job 1210: Subtask second_task
[2024-04-08T05:06:50.612+0000] {task_command.py:388} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-14T00:00:00+00:00 [running]> on host 3dc1147ef06e
[2024-04-08T05:06:50.696+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Jiseungmin
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-14T00:00:00+00:00
[2024-04-08T05:06:50.697+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-04-08T05:06:50.699+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello i am task2 and this is the task2 task!']
[2024-04-08T05:06:50.708+0000] {subprocess.py:86} INFO - Output:
[2024-04-08T05:06:50.710+0000] {subprocess.py:93} INFO - Hello i am task2 and this is the task2 task!
[2024-04-08T05:06:50.710+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-08T05:06:50.743+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211114T000000, start_date=20240408T050650, end_date=20240408T050650
[2024-04-08T05:06:50.830+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-04-08T05:06:50.850+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
